src/app/type/ProductList.js

```diff
@@ -118,7 +118,11 @@ export const ProductType = PropTypes.shape({
     special_price: PropTypes.number,
     thumbnail: PropTypes.shape({ url: PropTypes.string }),
     thumbnail_label: PropTypes.shape({ label: PropTypes.string }),
-    tier_prices: PropTypes.string,
+    tier_prices: PropTypes.arrayOf(PropTypes.shape({
+        value: PropTypes.number.isRequired,
```

Comment:
Do not put `isRequired` here, this is overall type, people might request those, might not :)